src/js/components/leftPanel/layersPanel/LayerTransparencySlider.tsx

```diff
@@ -19,6 +19,13 @@ const LayerTransparencySlider = (
 
   const handleOpacityChange = (eventValue: any): void => {
     mapController.setLayerOpacity(layerID, eventValue, sublayer, parentID);
+
+    if (
+      layerID.includes('VIIRS_ACTIVE_FIRES') ||
+      layerID.includes('MODIS_ACTIVE_FIRES')
+    ) {
+      mapController.updateMODISorVIIRSOpacity(layerID, eventValue);
+    }
```

Comment:
this needs to be `if` `else`, right now even if we are dealing with fires layers, we are still kicking off setlayerOpacity above, that causes perf issues and errors.